.github/workflows/next.yml

```diff
@@ -10,11 +10,19 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        php: ["8.0", "8.1"]
-        drupal: ["9.4", "9.5", "10.0"]
+        # Supported PHP versions: https://www.drupal.org/docs/getting-started/system-requirements/php-requirements
+        php: ["8.0", "8.1", "8.2"]
+        # Supported Drupal versions: https://www.drupal.org/project/drupal
+        drupal: ["9.5", "10.0", "10.1"]
         exclude:
-          - php: "8.0"
-            drupal: "10.0"
+          - drupal: "9.5"
+            php: "8.1"
```

Comment:
Let's keep Drupal 9.5 & PHP 8.1. We're still using this combo on a handful of projects.

Reply:
Good catch. Drupal's PHP requirement docs say that combo is supposed. https://www.drupal.org/docs/getting-started/system-requirements/php-requirements